### ISSUE TITLE
Scripts

### ISSUE THREAD

**Issue by byelik** [Brackeys/NavMesh-Tutorial#4]
Hi, thanks for you tutorials. I've downloaded the source code, but there are not all scripts in the repo. Is it possible to get all needed scripts for these tutorials?
Thanks in advance!

**Comment by sqr1be** [comment]
yeah, not helpful. The playercontroller script i copied verbatim from the youtube video, and my little agent doesn't do any movement really. if i click on the agent it'self it seems to move a little. like, micro units.

**Comment by MTF-TAU-5** [comment]
Did you do it in game mode?

**Comment by shrivaths16** [comment]
Adding the **box collider** to the ground is actually **necessary** and not optional. We need a collider to detect the mouse clicks. 
Clicking on the player moves it because it has a capsule collider on it.

**Comment by arch-kobra1985** [comment]
"SetDestination" can only be called on an active agent that has been placed on a NavMesh.
UnityEngine.AI.NavMeshAgent:SetDestination (UnityEngine.Vector3)
playercontroler:Update () (at Assets/playercontroler.cs:21)
 



is what it said for me when I added a box solider to the ground

**Comment by arch-kobra1985** [comment]
<img width="1440" alt="Screen Shot 2021-07-16 at 8 25 21 PM" src="https://user-images.githubusercontent.com/76260622/126021279-351b74a1-5d39-4cae-8dfd-d7ed48d621b8.png">

**Comment by arch-kobra1985** [comment]
I was in play mode when it happened

**Comment by jeddles88** [comment]
theres a few issues in these examples
example one needs a box collider to work

example 3 needs more code in start on player controller so that when the player spawns it has a meshagent and a camera.
   void Start()
    {
        agent = gameObject.GetComponent<NavMeshAgent>();
        cam = Camera.main;
    }
